### ISSUE TITLE
Modal is Buggy

### ISSUE THREAD

**Issue by kunaljaypatel** [creativetimofficial/ct-now-ui-dashboard-pro-react#17]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [yes ] I am running the latest version
- [yes ] I checked the documentation and found no answer
- [ yes] I checked to make sure that this issue has not already been filed
- [ yes] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

The Modal appears in the correct location on every presentation.

# Current Behavior

Sometimes the Modal does not appear in the correct location.
(https://imgur.com/a/53GNcAr)

# Failure Information (for bugs)

The Modal in Now-Ui-Dashboard-react is buggy on Safari. A portion of it sometimes gets cut off in the Safari Browser on presentation. 

## Steps to Reproduce

Use Safari Browser.
1. Navigate to the premade Notifications page.
2. Click "Notice Modal", then click "small alert modal". Keep clicking, opening, and repeating this process until you notice that one of the modals doesn't appear properly and gets stuck while cut off. 

## Context

Occurs on Safari. The code is the default code that comes with the latest version of Now-UI Dashboard Pro React. 

* Device: Macbook
* Operating System: Mac OS Mojave
* Browser and Version: Safari Version 12.0.1

## Failure Logs

N/A

**Comment by einazare** [comment]
Hello again, @kunaljaypatel ,

I'll take a look at this in a few days.
Right now I am away with some personal matters.

Best,
Manu

**Comment by kunaljaypatel** [comment]
Hey @EINazare ,

Have you had a chance to look at it? 

I tried debugging it a few days ago, and I was able to figure out a solution. If you turn                  fade={false} in the Modal, the Modal doesn't get "stuck". I assume the animation is just bugged on Safari with now-ui-dashboard-pro-react.

I went to[ ReactStrap](https://reactstrap.github.io/components/modals/) and tried their modals (on Safari). It works consistently there with  fade={true}. I'm not sure what's causing the animation to get "stuck" on now-ui-dashboard-pro-react . 

Let me know if you have any issues reproducing the bug.

Thanks,

Kunal

**Comment by einazare** [comment]
Hello again @kunaljaypatel ,

Sorry for this late response.
Last few weeks have been a chaos.
I've tested it and haven't been able to reproduce your issue.
I've actually tested on the same device as you.
<img width="280" alt="screen shot 2018-12-11 at 10 52 06 am" src="https://user-images.githubusercontent.com/26245512/49788750-da739480-fd32-11e8-9925-c120cf2873e6.png">

Now sure how to help you.

Best,
Manu
